rules/ref/path/dev-shm.yara

```diff
@@ -15,7 +15,7 @@ rule dev_shm_file : high {
     hash_2023_BPFDoor_8b9d = "8b9db0bc9152628bdacc32dab01590211bee9f27d58e0f66f6a1e26aea7552a6"
     hash_2023_OK_ad69 = "ad69e198905a8d4a4e5c31ca8a3298a0a5d761740a5392d2abb5d6d2e966822f"
   strings:
-    $ref = /\/dev\/shm\/[\%\w\.]{2,64}/
+    $ref = /\/dev\/shm\/[\%\w\.\-\/]{2,64}/
```

Comment:
This was cutting off the full result (e.g., `/dev/shm/.vigpg-XXXXXXXXXXXX` was being truncated to `/dev/shm/.vigpg`).